cms/static/js/spec/video/transcripts/editor_spec.js

```diff
@@ -132,6 +143,16 @@ function($, Backbone, _, Utils, Editor, MetadataView, MetadataModel, MetadataCol
                     type: MetadataModel.GENERIC_TYPE,
                     value: 'OEoXaMPEzfM'
                 },
+                videoIDEntry = {
+                    default_value: 'test default value',
+                    display_name: 'Video ID',
+                    explicitly_set: true,
+                    field_name: 'edx_video_id',
+                    help: 'Specifies the video ID.',
+                    options: [],
+                    type: MetadataModel.GENERIC_TYPE,
+                    value: 'advanced tab video id'
+                },
```

Comment:
Can we define value of videoIDEntry at one place instead of duplicating the same value at different places ?